ee/dtr/user/manage-images/sign-images/trust-with-remote-ucp.md

```diff
@@ -243,6 +246,14 @@ This means that the image was signed correctly, however the user who signed the
 image does not meet the signing policy in cluster 2. This could be because you
 signed the image with the wrong user keys.
 
+#### DTR URL must be a registered trusted registry
+
+```
+Error response from daemon: dtr.example.com must be a registered trusted registry. See 'docker run --help'.
```

Comment:
I would remove the 'See docker run --help' bit. That is a second line in the error message, and is relevant to how the container was started not related to the trust error message. Took me a double take to work out where it came from :D

Reply:
Yikes, so picky. But will do!

Reply:
Hmmm, that second bit _is_ there and isn't on a separate line.

The `docker` cli templatizes the error, and puts "Error response from daemon: " at the front, and "See 'docker run --help'" at the end. In the middle is the actual text from the daemon, in this case UCP is acting like the docker daemon and generates that part of the error.

Since the "See 'docker run --help'" portion of the error message is 100% inline and isn't separated by a newline, I'm inclined to keep it in the docs since that's the actual text of the error message that will be seen by the user.

Reply:
@programmerq Ahh ok, I didn't realise that. Happy to have the `docker run --help` put back in, to help customers find an exact match for the error message. I'll do a quick PR. 

In reality that "docker run --help" isn't going to help them very much in troubleshooting that particular error. But that's a whole different can of worms.... :D